src/components/views/elements/LazyRenderList.tsx

```diff
@@ -51,20 +50,55 @@ class ItemRange {
         );
     }
 
-    totalSize() {
+    public totalSize(): number {
         return this.topCount + this.renderCount + this.bottomCount;
     }
 }
 
+interface IProps<T extends unknown> {
```

Comment:
Tbh, I am quite unsure about the typing here

Reply:
Should be able to just use `<T>` and make `LazyRenderList` also generic to pass through T, in theory. Might need to set a default on `LazyRenderList` though: `LazyRenderList<T = any> extends React.Component<IProps<T>, IState>`

Reply:
Thanks!